Resources/config/schema/doctrine-1.0.xsd

```diff
@@ -37,6 +37,7 @@
         <xsd:attribute name="logging" type="xsd:string" default="false" />
         <xsd:attribute name="profiling" type="xsd:string" default="false" />
         <xsd:attribute name="server-version" type="xsd:string" />
+        <xsd:attribute name="use-savepoints" type="xsd:boolean" />
```

Comment:
Why? Drivers don't have anything related to save point configuration in them. The savepoint stuff is in the connection object itself.